src/Business Foundation/App/NoSeries/src/Single/NoSeriesImpl.Codeunit.al

```diff
@@ -141,7 +141,7 @@ codeunit 304 "No. Series - Impl."
         if not LineFound then
             LineFound := NoSeriesLine.FindLast();
 
-        if LineFound and NoSeries.MayProduceGaps(NoSeriesLine) then begin
+        if LineFound and NoSeriesLine.Open and NoSeries.MayProduceGaps(NoSeriesLine) then begin
             NoSeriesLine.Validate(Open);
             if not NoSeriesLine.Open then
                 NoSeriesLine.Modify(true);
```

Comment:
@grobyns Could you review, why are we doing a modify within a get procedure? I'm not familiar with `No Series` in general.